patches/@shopify+flash-list+1.6.3.patch

```diff
@@ -0,0 +1,1360 @@
+-
+-  - https://github.com/Shopify/flash-list/pull/257
+-
+-- initialScrollIndex, scrollTo methods will now account for size of header
+-
+-  - https://github.com/Shopify/flash-list/pull/194
+-
+-- Added a new mock for easier testing of components with `FlashList`
+-  - https://github.com/Shopify/flash-list/pull/236
+-
+-## [0.4.0] - 2022-03-23
+-
+-- Add support for layout animations
+-
+-  - https://github.com/Shopify/flash-list/pull/183
+-
+-- Suppress recyclerlistview's bounded size exception for some missing cases.
+-
+-  - https://github.com/Shopify/flash-list/pull/192
+-
+-- Expose reference to recyclerlistview and firstItemOffset
+-
+-  - https://github.com/Shopify/flash-list/pull/217
+-
+-- recyclerlistview upgraded to v3.1.0-alpha.9
+-  - https://github.com/Shopify/flash-list/pull/227
+-
+-## [0.3.3] - 2022-03-16
+-
+-- Prevent implicit scroll to top on device orientation change
+-- Change recyclerlistview's bounded size exception to a warning
+-  - https://github.com/Shopify/flash-list/pull/187
+-
+-## [0.3.2] - 2022-03-15
+-
+-- Minor changes
+-
+-## [0.3.1] - 2022-03-15
+-
+-- Revert react-native-safe-area upgrade and minSdkVersion bump
+-  - https://github.com/Shopify/flash-list/pull/184
+-
+-## [0.3.0] - 2022-03-15
+-
+-- Fixed untranspiled library code by enforcing stricter TS rules.
+-  - https://github.com/Shopify/flash-list/pull/181
+-
+-## [0.2.4] - 2022-03-14
+-
+-- Added `onLoad` event that is called once the list has rendered items. This is required because FlashList doesn't render items in the first cycle.
+-  - https://github.com/Shopify/flash-list/pull/180
+-
+-## [0.2.3] - 2022-03-10
+-
+-- Fixing publish steps for transpiled code
+-  - https://github.com/Shopify/flash-list/pull/150
+-
+-## [0.2.2] - 2022-03-10
+-
+-- Fixing publish steps for transpiled code
+-  - https://github.com/Shopify/flash-list/pull/149
+-
+-## [0.2.1] - 2022-03-09
+-
+-- Bug fix for style and last separator
+-  - https://github.com/Shopify/flash-list/pull/141
+-
+-## [0.2.0] - 2022-03-08
+-
+-- Rename the component from `RecyclerFlatList` to `FlashList`
+-  - https://github.com/Shopify/flash-list/pull/140
+-
+-## [0.1.0] - 2022-03-02
+-
+-- Initial release
+diff --git a/node_modules/@shopify/flash-list/RNFlashList.podspec b/node_modules/@shopify/flash-list/RNFlashList.podspec
+index 38ff029..a749f6c 100644
+--- a/node_modules/@shopify/flash-list/RNFlashList.podspec
++++ b/node_modules/@shopify/flash-list/RNFlashList.podspec
+@@ -9,14 +9,20 @@ Pod::Spec.new do |s|
+   s.homepage         = package['homepage']
+   s.license          = package['license']
+   s.author           = package['author']
+-  s.platforms        = { :ios => '11.0', :tvos => '12.0' }
+   s.source           = { git: 'https://github.com/shopify/flash-list.git', tag: "v#{s.version}" }
+   s.source_files     = 'ios/Sources/**/*'
+   s.requires_arc     = true
+   s.swift_version    = '5.0'
++  s.pod_target_xcconfig = { 'OTHER_SWIFT_FLAGS' => '-D RCT_NEW_ARCH_ENABLED', }
```

Comment:
looks like some of the upstream PRs have been closed. going to link https://github.com/Shopify/flash-list/pull/931 in the PR description for this.